src/WordPress/JsonMapper/Evaluators/NamespaceResolver.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace WordPress\JsonMapper\Evaluators;
+
+use PhpParser\NodeTraverser;
```

Comment:
This package is 250kb zipped. It's fine for this PR but let's explore switching to something lightweight like `token_get_all()` before shipping v1 of this library. Would you please start an issue to track that?

Reply:
This could come handy: https://gist.github.com/Zeronights/7b7d90fcf8d4daf9db0c

Reply:
Great point. Will monitor in issue #70